src/MoBi.Presentation/MenusAndBars/ContextMenus/ContextMenuForSimulation.cs

```diff
@@ -204,6 +205,7 @@ private IMenuBarItem createDeleteItem(IMoBiSimulation simulation)
       private IMenuBarItem createDeleteAllResultsItem(IMoBiSimulation simulation)
       {
          return CreateMenuButton.WithCaption(AppConstants.MenuNames.DeleteAllResults)
+            .WithEnabled(!_simulationRunner.IsSimulationRunning(simulation))
```

Comment:
What about adding an  `IsSimulationIdle` or sthg like this becuase all the time we are doing !xxx.IsSimulationRunning`? Would it maake sense?